### PR TITLE
Add ERNIE 3.0 Medium CLUE result

### PR DESCRIPTION
**Pull request by LiuChiachi**
### PR types

New features
### PR changes

Models
### Description

Add ERNIE 3.0-Medium CLUE result

### CODE REVIEW

File: examples/benchmark/clue/README.md [hunk 1]
```diff
@@ -9,11 +9,12 @@
...
-| RoBERTa-wwm-ext       | 74.75     | 58.08     | 61.22     | 81.66     | 77.25     | 88.55       | 81.63     | 68.69/88.75     | 83.41     | 67.69         | 74.42     |
+| Model                 | AVG   | AFQMC     | TNEWS     | IFLYTEK   | CMNLI     | OCNLI     | CLUEWSC2020 | CSL       | CMRC2018        | CHID      | C<sup>3</sup> |
+| --------------------- | ----- | --------- | --------- | --------- | --------- | --------- | ----------- | --------- | --------------- | --------- | ------------- |
+| RoBERTa-wwm-ext-large | 76.54 | 75.32     | 59.33     | 62.33     | 83.87     | 78.81     | 90.79       | 83.37     | 70.58/89.82     | 85.72     | 75.26         |
+| ERNIE-3.0-large       | 78.71 | **77.36** | **60.21** | **62.75** | **85.06** | **82.14** | **91.12**   | **84.23** | **72.76/91.87** | **86.84** | **84.67**     |
+| RoBERTa-wwm-ext       | 74.42 | 74.75     | 58.08     | 61.22     | 81.66     | 77.25     | 88.55       | 81.63     | 68.69/88.75     | 83.41     | 67.69         |
+| ERNIE-3.0-tiny        | 72.86 | 75.35     | 57.45     | 60.18     | 81.16     | 77.19     | 79.11       | 81.93     | 65.83/87.29     | 80.00     | 70.36         |
```
Comment (ZeyuChen) [inline examples/benchmark/clue/README.md]: ERNIE 3.0-Medium

ERNIE 3.0-Large

ERNIE 3.0-Base

**Review by ZeyuChen — COMMENTED** [review]
命名体系修改一下，Tiny改为Medium

**Review by tianxin1860 — DISMISSED** [review]
LGTM

**Review by tianxin1860 — APPROVED** [review]
LGTM

**Pull request merged**